unlock-protocol-com/src/components/icons/GrossNetworkIcons.tsx

```diff
@@ -0,0 +1,191 @@
+      </svg>
+    </IconBase>
+  )
+}
+
+export function BSCIcon(props: IconBaseProps) {
+  return (
+    <IconBase {...props}>
+      <svg
+        width="41"
+        height="40"
+        viewBox="0 0 41 40"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <rect x="0.5" width="40" height="40" rx="20" fill="#F0B90B" />
+        <g clipPath="url(#clip0_8124_502)">
+          <path
+            d="M20.5514 7.04357L23.7753 10.3451L15.6574 18.463L12.4335 15.2391L20.5514 7.04357Z"
+            fill="white"
+            stroke="white"
+            strokeWidth="0.17782"
+          />
+          <path
+            d="M25.4454 11.9378L28.6693 15.2393L15.6574 28.2512L12.4335 25.0274L25.4454 11.9378Z"
+            fill="white"
+            stroke="white"
+            strokeWidth="0.17782"
+          />
+          <path
+            d="M10.7648 16.832L13.9887 20.1336L10.7648 23.3574L7.541 20.1336L10.7648 16.832Z"
+            fill="white"
+            stroke="white"
+            strokeWidth="0.17782"
+          />
+          <path
+            d="M30.3397 16.832L33.5635 20.1336L20.5516 33.1455L17.3278 29.9216L30.3397 16.832Z"
+            fill="white"
+            stroke="white"
+            strokeWidth="0.17782"
+          />
+        </g>
+        <defs>
+          <clipPath id="clip0_8124_502">
+            <rect
+              width="26.1333"
+              height="26.2118"
+              fill="white"
+              transform="translate(7.48981 6.98981)"
+            />
+          </clipPath>
+        </defs>
+      </svg>
+    </IconBase>
+  )
+}
+
+export function CeloIcon(props: IconBaseProps) {
+  return (
+    <IconBase {...props}>
+      <svg
+        width="41"
+        height="40"
+        viewBox="0 0 41 40"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <circle cx="20.5" cy="20" r="20" fill="#F8F9F9" />
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M16.1 29.4287C19.9818 29.4287 23.1286 26.2819 23.1286 22.4001C23.1286 18.5183 19.9818 15.3715 16.1 15.3715C12.2182 15.3715 9.07143 18.5183 9.07143 22.4001C9.07143 26.2819 12.2182 29.4287 16.1 29.4287ZM16.1 32.0001C21.4019 32.0001 25.7 27.702 25.7 22.4001C25.7 17.0982 21.4019 12.8001 16.1 12.8001C10.7981 12.8001 6.5 17.0982 6.5 22.4001C6.5 27.702 10.7981 32.0001 16.1 32.0001Z"
+          fill="#FBCC5C"
+        />
+        <g>
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M20.8999 24.6286C24.7816 24.6286 27.9284 21.4818 27.9284 17.6C27.9284 13.7182 24.7816 10.5714 20.8999 10.5714C17.0181 10.5714 13.8713 13.7182 13.8713 17.6C13.8713 21.4818 17.0181 24.6286 20.8999 24.6286ZM20.8999 27.2C26.2018 27.2 30.4999 22.9019 30.4999 17.6C30.4999 12.2981 26.2018 8 20.8999 8C15.5979 8 11.2999 12.2981 11.2999 17.6C11.2999 22.9019 15.5979 27.2 20.8999 27.2Z"
+            fill="#35D07F"
+          />
+        </g>
+      </svg>
+    </IconBase>
+  )
+}
```

Comment:
We actually import crypto-icon package in unlock-app for exactly this. That supports all tokens, not just these. 

Can you please add it to the site and use it here? That would be save the boilerplate code.

Reply:
resolved